src/Control/Monad/Aff.js

```diff
@@ -96,6 +96,26 @@ exports._forkAff = function (nonCanceler, aff) {
   };
 }
 
+exports._forkAll = function (nonCanceler, foldl, affs) {
+  var voidF = function(){};
+
+  return function(success, error) {
+    foldl(function(_) {
+      return function(aff) {
+        aff(voidF, voidF);
+      };
+    })({})(affs);
+
+    try {
+      success({});
+    } catch(e) {
+      error(e);
+    }
+
+    return nonCanceler;
```

Comment:
It would be nice to be able to cancel these by propagating the canceling behavior into the forked computations, rather than returning a non-canceler here.